client/my-sites/site-settings/navigation.jsx

```diff
@@ -28,6 +28,7 @@ export class SiteSettingsNavigation extends Component {
 		const { translate } = this.props;
 		return {
 			general: translate( 'General', { context: 'settings screen' } ),
+			performance: translate( 'Performance', { context: 'settings screen' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Performance' )` *ES Score: 22*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/performance-settings-tab)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.